jbpm-designer-backend/src/main/java/org/jbpm/designer/web/preprocessing/impl/JbpmPreprocessingUnit.java

```diff
@@ -145,33 +147,118 @@
     private VFSService vfsService;
     private boolean includeDataObjects;
 
-    @Inject
-    protected Event<DesignerWorkitemInstalledEvent> workitemInstalledEventEvent;
-
-    @Inject
-    protected Event<NotificationEvent> notification;
-
     @Inject
     private POMService pomService;
 
-    @Inject
-    private ProjectService<? extends Project> projectService;
-
     @Inject
     private MetadataService metadataService;
 
     @Inject
     private DefaultDesignerAssetService defaultDesignerAssetService;
 
-    @Inject
-    KieProjectService kieProjectService;
-
     @Inject
     private LRUBuilderCache builderCache;
 
     public JbpmPreprocessingUnit() {
     }
 
+    protected static String readFile(String pathname) throws IOException {
```

Comment:
Any reason for hand rolling custom "read file contents as String" when we could rely on well tested implementations from commons-io [IOUtils](https://commons.apache.org/proper/commons-io/javadocs/api-release/org/apache/commons/io/IOUtils.html) (which this class is importing anyway)?

Reply:
If you look lower down, this method was simply moved. As much as I agree that we should just delegate the functionality to `IOUtils`, we should try and minimise the amount of non-essential changes made in these PRs.

This is already a huge change-set (the entire set of PRs, that is), with many moving parts. It's going to take a long time to review/merge if we allow scope creep to include fixing all the old mistakes we find.

With that in mind, I would suggest that you try and avoid reviewing code that has just been moved or renamed unless you have reason to believe that other changes have made that code *less* correct.